blueprints/app/files/package.json

```diff
@@ -72,7 +72,7 @@
     "@typescript-eslint/parser": "^6.21.0<% } %>",
     "broccoli-asset-rev": "^3.0.0",
     "concurrently": "^8.2.2",
-    "ember-auto-import": "^2.7.3",
+    "ember-auto-import": "^2.7.4",
     "ember-cli": "~<%= emberCLIVersion %>",
     "ember-cli-app-version": "^6.0.1",
```

Comment:
v7 resolves upcoming deprecations, and drops < node 18 https://github.com/ember-cli/ember-cli-app-version/releases/tag/v7.0.0